healthcheck/healthcheck.go

```diff
@@ -58,6 +45,9 @@ func ExecuteOnAPI(c *gin.Context) {
 
 	if result.Status == Unhealthy {
 		logInstance("ExecuteOnAPI").ErrorBasicWithContent("Healthcheck is Unhealthy", "HealthCheck", result)
+
```

Comment:
Poderia ter uma lógica em cima apenas do status, evitando duplicar a instrução `c.JSON()`
```suggestion
      status := http.StatusOK
	healtcheck := createHealthCheck()
	result := healtcheck.Execute()

	if result.Status == Unhealthy {
		logInstance("ExecuteOnAPI").ErrorBasicWithContent("Healthcheck is Unhealthy", "HealthCheck", result)
		status = http.StatusServiceUnavailable
	}

	c.JSON(status, newHealthCheckResponse(&result))
```